benchmarks/status.py

```diff
@@ -64,3 +64,14 @@ def add_ignore_rules(path, number):
     def setup(self):
         super().setup()
         self.add_ignore_rules(self.test_directory.name, 30)
+
+
+class DVCIgnoreSubrepoBench(DVCIgnoreBench):
```

Comment:
Do we want the `DVCIgnoreSubrepoBench` to include ignore rules from `DVCIgnoreBench`? They will be there if we inherit after `DVCIgnoreBench`. Maybe we should inherit after `DVCStatusBench`? That way we will be observing only the influence of subrepos.